### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by dimitris90s** [nixgates/plugin.video.seren#87]
Hello Nix,

I would like to make a suggestion if it possible.
There are many non English people who using your addon and it would be great if you can add a language menu to movies and tv shows sections like original Exodus.
It would be really helpfull.

Thanks for this great addon!

**Comment by cronofx** [comment]
Hi guys I new on GitHub not quite there yet , been useing seren for a quite bit now and I think you guys made an amazing add-on  ,  there is space to improve though any chance to make track.tv   make sync on start up ?? Would save   effort of doing it manually everytime cheers.

**Comment by ShahidSaud** [comment]
Hey Nix,
I agree with dimitris90s. Once I started using Seren I haven't looked on for any other addon and the addon is just amazing. Only thing missing is the Language option which dimitris90s mentioned, I rarely goto other addon when I need to watch French or German movies or series.

Please plan to include this option in your addon or let us know when this will be available as an update? Yes, this is an enhancement but I am sure many non-English speakers will be happy to use this addon further.

Appreciate for this addon and the options it has though!

Regards,
Shahid

**Comment by drinfernoo** [comment]
1.4.6 has added translation support for most of the add-on strings. It's now possible to do these translations!

**Comment by guglicap** [comment]
Hello, there I'd be more than happy to provide translated strings for Italian, if needed. Let me know the way I should go about it.

**Comment by nixgates** [comment]
@guglicap Seren now uses your local language in Kodi when fetching meta from providers.

If you're looking to add a language to Seren feel free to raise an issue for it and I'll help you get it done :)

**Comment by ShahidSaud** [comment]
Hi @nixgates, its not the addon language we are talking about. Its the Movies/Series language we refer to. In popular addon's like Numbers, Exodus etc, once you navigate inside Movies or TVShows you have an option of New Movies, Popular etc there is one option called Languages which lists Movies/TvShows in different languages like Arabic, Bengali, Catalan, English, French, German, ...etc
We were hoping if you can include such feature in the Seren Addon.

Thanks and Warm regards,
Shahid

**Comment by nixgates** [comment]
@ShahidSaud Unless I'm getting this wrong, Seren now fills the metadata for shows/movies using the local language set in Kodi.

The New Shows endpoint also makes use your local language to include shows of your language. 

Are you after a specific menu to show shows that are released natively in your language?

**Comment by ShahidSaud** [comment]
hey @nixgates , good morning... I am sorry I think I am not able to explain it to you. Let me try again with an example :)

We have genres menu under Movies and Shows right... In Genres we have Action, Romance, Horror, Thriller etc. Most of the add-ons have Languages under Movies and Shows. Which has contents organized in different language not just in the user native language. For Example: I speak English and I want to stream a popular French TVShow in French Language right now in Seren what I do is. I go to search and find that show and stream its Episodes. If there is a section under TVShows Languages and French, I will go directly under that section and stream the particular show from there.

So the request is to add a Language section under Movies or TVShows which has contents arranges in different languages.
